circle2/setup-java-service.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+# Sets up docker, JCE unlimited, and maven
+#
+# Non Circle Provided Input Variables:
+#   DOCKER_EMAIL: quay.io email (can be anything)
+#   DOCKER_USER: quay.io username
+#   DOCKER_PASS: quay.io encrypted password
+#   MVN_SETTINGS_XML: contents of ~/.m2/settings.xml
+
+set -e
+
+setup-docker.sh
+#setup-jce.sh
```

Comment:
we don't need JCE at all in the circle2 containers?

Reply:
it appears to be set up by default!